src/plugins/expressions/common/execution/execution.test.ts

```diff
@@ -666,7 +666,7 @@ describe('Execution', () => {
           params: { debug: true },
         });
         execution.start(0);
-        await execution.result;
+        await execution.result.toPromise();
```

Comment:
can we add some tests that will actually check behaviour with observables ?
- input being an observable
- one of the functions returning an observable
- one of the arguments to function being an observable
- testing of partial results
- testing of final result
- test for failiure

Reply:
- input emits once
- input emits multiple times
- input errors
- input completes
- argument emits once
- argument emits multiple times
- multiple arguments emit multiple times
- arguments error
- arguments complete